pkg/cmd/pipeline/start_test.go

```diff
@@ -454,6 +454,22 @@ func TestPipelineStart_ExecuteCommand(t *testing.T) {
 			wantError:  false,
 			goldenFile: true,
 		},
+		{
+			name: "Dry Run with use param defaults and output=json",
+			command: []string{"start", "test-pipeline",
+				"-s=svc1",
+				"-r=source=scaffold-git",
+				"--use-param-defaults",
+				"-l=jemange=desfrites",
+				"-n", "ns",
+				"--dry-run",
+				"--output", "json",
+			},
+			namespace:  "",
+			input:      cs2,
+			wantError:  false,
+			goldenFile: true,
+		},
```

Comment:
Can you also add another test for the case where a value is overrode using `-p` that has a default, but `--use-param-defaults` is also used to still appropriately use the defaults of other values?

Reply:
Sure will add it.